src/middlewares/userMiddlware.js

```diff
@@ -41,6 +41,19 @@ const User = {
     next();
   },
 
+  async getUserById(req, res, next) {
+    const { id } = req.params;
+    const user = await db.Users.findOne({
```

Comment:
@ great job urban but at least export this to `user.service.js` since its one of the `CRUD` operations

Reply:
great!

Reply:
Luckily that service is already there

Reply:
cool